### PR TITLE
Update manage-addins-in-the-admin-center.md

### PR DESCRIPTION
**Pull request by v-padum**
Although the link for Services & Add-ins is correct and still points to the right place, the text no longer represents how the UI element is titled. Rather, it's now part of Settings > Org settings and Settings > Integrated apps based on which specific setting we're talking about.

Because of the new admin center, the experience to remove add-ins is also extremely convoluted. Instructions updated in this PR.

The note can also be removed because there is no longer an alternate UI in the works (as far as I understand it) and the production version has been rolled out everywhere in WW.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -46,25 +46,23 @@ Consider deleting an add-in if no one is using it anymore. For example, turning
 
 You can also delete an add-in that was deployed.
 
-1. In the admin center, go to the **Settings** > **Services & add-ins** page.
+1. In the admin center, go to the **Settings** > **Integrated apps** page.
 
-     > [!NOTE]
-    > The admin center is getting updated to deployment experience with Integrated Apps . If you don't see the above steps, go to Centralized Deployment section by going to **Settings** > **Integrated apps**. On the top of the **Integrated apps** page, choose **Add-ins**.
+2. Select the deployed add-in and click the **Configuration** tab
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
2. Select the deployed add-in and then select the **Configuration** tab.
```

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 2]
```diff
@@ -46,25 +46,23 @@ Consider deleting an add-in if no one is using it anymore. For example, turning
...
 
-2. Select the deployed add-in.
+3. In the **Configuration** pane go to **Advanced Settings** > **Add-ins**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
3. In the **Configuration** pane, go to **Advanced Settings** > **Add-ins**.
```

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 3]
```diff
@@ -46,25 +46,23 @@ Consider deleting an add-in if no one is using it anymore. For example, turning
...
+
+4. Select the add-in from the list again
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
4. Select the add-in from the list again.
```

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 4]
```diff
@@ -46,25 +46,23 @@ Consider deleting an add-in if no one is using it anymore. For example, turning
...
 
-3. Click on **Delete Add-In**. Remove the Add-in button on the bottom right corner.
+5. Click on **Remove Add-In**. Remove the Add-in button on the bottom right corner.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
5. Choose **Remove Add-In**. Remove the Add-in button on the bottom right corner.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-padum, could you please commit my suggested changes if you agree with them? Many thanks.

**Comment by v-padum** [comment]
Done, perfect and thanks!
________________________________

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@tiburd Please resolve this conflict. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**